crates/eips/src/eip4844/env_settings.rs

```diff
@@ -43,25 +42,7 @@ impl EnvKzgSettings {
     #[inline]
     pub fn get(&self) -> &KzgSettings {
         match self {
-            Self::Default => {
-                let load = || {
-                    KzgSettings::load_trusted_setup(&G1_POINTS.0, &G2_POINTS.0)
-                        .expect("failed to load default trusted setup")
-                };
-                #[cfg(feature = "std")]
-                {
-                    use once_cell as _;
-                    use std::sync::OnceLock;
-                    static DEFAULT: OnceLock<KzgSettings> = OnceLock::new();
-                    DEFAULT.get_or_init(load)
-                }
-                #[cfg(not(feature = "std"))]
-                {
-                    use once_cell::race::OnceBox;
-                    static DEFAULT: OnceBox<KzgSettings> = OnceBox::new();
-                    DEFAULT.get_or_init(|| alloc::boxed::Box::new(load()))
-                }
-            }
+            Self::Default => c_kzg::ethereum_kzg_settings(0),
```

Comment:
this ckzg api is incredibly weird, because this allows providing an argument which is then ignored the second time this function is called because the kzgsettings are cached.
however, for this crate this is perfectly fine

Reply:
Ref https://github.com/ethereum/c-kzg-4844/blob/dffa18ee350aeef38f749ffad24a27c1645fb4f8/src/setup/setup.c#L401-L407